regression/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(test_pl_path "${CMAKE_CURRENT_SOURCE_DIR}/test.pl")
 macro(add_test_pl_profile name cmdline flag profile)
     add_test(
         NAME "${name}-${profile}"
-        COMMAND ${test_pl_path} -e -p -c "${cmdline}" ${flag} ${ARGN}
+        COMMAND perl ${test_pl_path} -e -p -c "${cmdline}" ${flag} ${ARGN}
```

Comment:
The issue here is that apparently on windows the shebang thing isn't really a thing. It's possible to hack something similar together by associating file endings with runners which is what python does, but apparently this doesn't work for perl.

Commit message should mention this (4edec5a5cf78aaaece8367d30b3ccb38d18ad7bc)